<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="fr_FR">
<context>
    <name>QObject</name>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="29"/>
        <source>&lt;strong style=&quot;color:red&quot;&gt;Username or password invalid. Try again.&lt;/strong&gt;</source>
        <translation>&lt;strong style=&quot;color:red&quot;&gt;Nom d&apos;utilisateur ou mot de passe invalide. Veuillez réessayer.&lt;/strong&gt;</translation>
    </message>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="33"/>
        <source>Show password</source>
        <translation>Voir le mot de passe</translation>
    </message>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="35"/>
        <source>Authorize</source>
        <translation>Autoriser</translation>
    </message>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="36"/>
        <source>No thanks</source>
        <translation>Non merci</translation>
    </message>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="43"/>
        <source>Username or email:</source>
        <oldsource>Username or email :</oldsource>
        <translation>Nom d&apos;utilisateur ou courriel :</translation>
    </message>
    <message>
        <location filename="ui/authentication/loginwidget.cpp" line="44"/>
        <source>Password:</source>
        <oldsource>Password :</oldsource>
        <translation>Mot de passe :</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="115"/>
        <location filename="ui/authentication/oauthwidget.cpp" line="130"/>
        <source>Error during the authentication process</source>
        <oldsource>Erreur dans le processus d&apos;authentification</oldsource>
        <translation>Erreur pendant la procédure d&apos;authentification</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="116"/>
        <source>Error while launching the authentication process. Authentication ended.</source>
        <oldsource>Erreur au lancement de l&apos;authentification. Authentification terminÃ©e.</oldsource>
        <translation>Erreur au lancement de la procédure d&apos;authentification. Authentification terminée.</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="127"/>
        <source>
End of the authentication process.</source>
        <oldsource>
Fin de l&apos;authentification.</oldsource>
        <translation>Fin de la procédure d&apos;authentification.</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="135"/>
        <source>
Would you like to resume the authentication process ?</source>
        <oldsource>
Voulez-vous recommencer l&apos;authentification ?</oldsource>
        <translation>Voulez-vous recommencer la procédure d&apos;authentification ?</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="138"/>
        <source>Hitch in the authentication process</source>
        <oldsource>ImprÃ©vu dans le processus d&apos;authentification</oldsource>
        <translation>Imprévu dans la procédure d&apos;authentification</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="150"/>
        <source>End of the authentication process</source>
        <oldsource>Fin du processus d&apos;authentification</oldsource>
        <translation>Fin de la procédure d&apos;authentification</translation>
    </message>
    <message>
        <location filename="ui/authentication/oauthwidget.cpp" line="151"/>
        <source>Authentication ended</source>
        <oldsource>Authentification terminée</oldsource>
        <translation>Authentification terminée</translation>
    </message>
</context>
</TS>
